### PR TITLE
.net guide new name

### PR DESCRIPTION
**Pull request by tdykstra**

### CODE REVIEW

File: docs/toc.md [hunk 1]
```diff
@@ -223,7 +223,7 @@
 ### [Documenting your code](csharp/codedoc.md)
 ### [Versioning](csharp/versioning.md)
 <!-- End of C# Concepts section -->
-<!--## [🔧 Using the .NET Compiler Platform](csharp/roslyn/index.md)-->
+<!--## [🔧 Using the .NET Compiler](csharp/roslyn/index.md)-->
```
Comment (rpetrusha) [inline docs/toc.md]: This is Roslyn; the official name of the technology is ".NET Compiler Platform".

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Roslyn can't be renamed, since it's the technology's official name. Otherwise, LGTM.

File: docs/toc.md (hunk 1 above)
Comment (tdykstra) [inline docs/toc.md]: Fixed, thanks.

Comment (tdykstra) [inline docs/toc.md]: Consistent word use will always be beyond our reach 😢

**Review by rpetrusha — APPROVED** [review]
LGTM, @tdykstra. This is ready to merge.

**Review by mairaw — APPROVED** [review]
LGTM. Thanks @tdykstra! Need to wait a couple of builds to finish.

**Pull request merged**